src/lib/Components/StickyTabPage/StickyTabScrollView.tsx

```diff
@@ -14,9 +14,6 @@ export const useStickyTabContext = () => {
   return useContext(StickyTabScrollViewContext)
 }
 
-// how fast you have to be scrolling up to show the header when not near the top of the scroll view
-const SHOW_HEADER_VELOCITY = 10
```

Comment:
This idea didn't pan out, or?

Reply:
It's better if it just appears when the user comes back to the top of the view.